slices/shared-mime-info.yaml

```diff
@@ -0,0 +1,25 @@
+package: shared-mime-info
+
+essential:
+  - shared-mime-info_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libglib2.0-0t64_core
+      - libstdc++6_libs
+      - libxml2_libs
+    contents:
+      /usr/bin/update-mime-database:
```

Comment:
can we also have a test for this pls?

Reply:
Thanks for catching that, ported the tests from [24.04](https://github.com/lengau/chisel-releases/blob/1947fcf209051e5eaaeccbe2efd7b7af33386de8/tests/spread/integration/shared-mime-info/task.yaml) in 6fc738e